aws/ecr/outputs.tf

```diff
@@ -77,3 +33,12 @@ output "ecr_repository_url_api" {
   description = "URL of the Forms API's ECR"
   value       = aws_ecr_repository.api.repository_url
 }
+
+output "ecr_repository_lambda_urls" {
```

Comment:
I like this map approach for the Lambda URLs and passing them around to different modules.

Reply:
Thanks, I got tired of dealing with 13 different outputs and inputs which would only grow over time as we potentially include more lambda functions.